crates/astria-bridge-withdrawer/src/withdrawer/batch.rs

```diff
@@ -0,0 +1,304 @@
+use std::time::Duration;
+
+use astria_core::{
+    primitive::v1::{
+        asset,
+        asset::Denom,
+        Address,
+    },
+    protocol::transaction::v1alpha1::{
+        action::{
+            BridgeUnlockAction,
+            Ics20Withdrawal,
+        },
+        Action,
+    },
+};
+use astria_eyre::eyre::{
+    self,
+    OptionExt,
+    WrapErr as _,
+};
+use ethers::types::{
+    TxHash,
+    U64,
+};
```

Comment:
I think it would be a bit more clear further down if these types weren't "used" and instead fully specify the type path below. I was confused about why `u64` was capitalized further down and it's good to have clarity on what type of txHash we are talking about (it's an EVM tx hash not a cometbft one)

Reply:
moved to the ethereum crate as this logic is eth-specific